plugins/generator-1.21.1/neoforge-1.21.1/procedures/entity_set_flying.java.ftl

```diff
@@ -1,4 +1,6 @@
-if (${input$entity} instanceof Player _player) {
+<@head>if (${input$entity} instanceof Player _player) {</@head>
 	_player.getAbilities().flying = ${input$condition};
+<@tail>
 	_player.onUpdateAbilities();
-}
\ No newline at end of file
+}
+</@tail>
```

Comment:
A new line here will prevent grouping:

<img width="300" height="392" alt="image" src="https://github.com/user-attachments/assets/1bba95a8-3487-4d9b-b196-6e69cc1ee161" />

Please double-check blocks to their heads and tails match 100%, otherwise we are losing potential of this feature